### DISCUSSION TITLE
Shorter settings menu sidebar width

### DISCUSSION THREAD

**Discussion by NeekoVN** [ppy/osu#25413] [Ideas] [answered]
How about shortening the width of sidebar in the settings menu so that it matches the width of the settings icon on the top left corner?
I mean it doesn't impact that much to the UI but it just feels kinda weird once you pay attention to it.
![image](https://github.com/ppy/osu/assets/76953348/41a2c230-884d-4c48-8385-31274e323995)

**Comment by Sclafus** [comment]
Honestly, I would like to see the exact opposite (making the settings button bigger)

**Comment by Joehuu** [comment, marked as answer]
The settings sidebar is always expanded now, so this is outdated now.

**Reply by NeekoVN** [reply]
Nice 👀
